### ISSUE TITLE
Docker container

### ISSUE THREAD

**Issue by ideologysec** [TheFrenchGhosty/TheFrenchGhostys-Ultimate-YouTube-DL-Scripts-Collection#48]
any plans to build a Docker container? would a pull request with a Dockerfile be accepted?

**Comment by TheFrenchGhosty** [comment]
I thought about it, but I don't really know how to handle it considering the number of scripts.

I guess a Docker can be made for each of them...

Feel free to open a PR I guess.

**Comment by ideologysec** [comment]
the easiest thing to do would be to build the container with ffmpeg, yt-dlp, and maybe even atomicparsley, plus your scripts, have the repo included, and the entrypoint be bash. that way the container would be interactive, and you could just the use the commands as normal. basically an easy packaging approach.

**Comment by TheFrenchGhosty** [comment]
This is the basic setup yes, but I think a Docker container should be able to run the script automatically when the Source - X.txt (mounted) file is edited.

**Comment by Stig124** [comment]
Hello, I can help

**Comment by s-kaczmarek** [comment]
Or rather use a cron inside container to run all scrips let's say each 5 min

**Comment by peterpepo** [comment]
Any news on this?
This would be absolutely amazing.
Unfortunately, I'm not knowledgeable enough to build the image by myself.

Cron/scheduler is good idea. Mount a volume from outside, to which would the image write downloads in same structure as scripts themselves, e.g. archivist/unique/ etc..

**Comment by signed-log** [comment]
If I have time, I'll work on it

**Comment by TheFrenchGhosty** [comment]
>Any news on this?

Not really, I can't think of a good/clean way to do it, and I personally don't want/need it.
